src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.test.tsx

```diff
@@ -295,9 +300,18 @@ describe('Financial Account Transactions', () => {
 
     it('should export CSV', async () => {
       const mockBlobUrl = 'blob:';
-      buildURI.mockReturnValue(mockBlobUrl);
+      (buildURI as jest.Mock).mockReturnValue(mockBlobUrl);
```

Comment:
I believe you can revert this change.

```suggestion
      buildURI.mockReturnValue(mockBlobUrl);
```